documentation/modules/overview/con-security-configuration-authorization.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// overview/assembly-security-overview.adoc
+
+// UserStory: Securing Kafka -- authorization
+
+[id="security-configuration-authorization_{context}"]
+= Authorization
+You can apply simple authorization to a Kafka cluster.
```

Comment:
the _simple_ is a name. I think we need to be careful how we present it. Either we should call it `SimpleACLAuthorizer` - which is the name - or leave out the simple. The simple suggests it is not good enough.

Reply:
Good point. I've reworded the section.